guide/general/priority_list.md

```diff
@@ -181,7 +189,16 @@ If you are using {{ site.data.talent.eb }} and {{ site.data.spell.primordial_wav
     1. {{ site.data.spell.frs }}
 - don't stress too much over gaming {{ site.data.talent.mote }}. It mostly happens naturally.
 
-
+### {{ site.data.talent.if }} with {{site.data.legendary.windspeaker}} equipped and {{site.data.talent.eb}} talented.
+-The priority with these conditions met is a little complex so this section can be used to help players learn and become more familiar with the setup.
+- Use {{site.data.talent.if}} on cooldown, or as higher priority if a mechanic will force you to move in the next 5 seconds.
+- {{site.data.talent.if}} charges should aim to be consumed before they expire, subject to the following priority:
+    -{{site.data.spell.lvb}} (without {{site.data.talent.mote}} active), {{site.data.talent.eb}} and {{site.data.spell.es}} have higher priority when {{site.data.talent.if}} buff duration is higher than the GCDs required to spend the remaining stacks.
```

Comment:
each bullet point here requires a space after the dash to be properly detected as a list